lib/compiler/webpack-compiler.ts

```diff
@@ -140,6 +140,7 @@ export class WebpackCompiler {
         return process.exit(1);
       }
       console.log(statsOutput);
+      compiler.close(() => {});
```

Comment:
Are you sure we want to close the compiler - even if it works in the watch mode?

Reply:
you are correct. should not close in watch mode... i'll update the PR